code/__DEFINES/radio.dm

```diff
@@ -81,6 +81,26 @@
 #define RADIO_KEY_DEN "j"
 #define RADIO_TOKEN_DEN ":j"
 
+#define RADIO_CHANNEL_SALVATORES "Salvatores"
+#define RADIO_KEY_SALVATORES "s"
+#define RADIO_TOKEN_SALVATORES ":s"
+
+#define RADIO_CHANNEL_BISHOPS "Bishops"
+#define RADIO_KEY_BISHOPS"b"
+#define RADIO_TOKEN_BISHOPS ":b"
+
+#define RADIO_CHANNEL_MORDINOS "Mordinos"
+#define RADIO_KEY_MORDINOS "m"
+#define RADIO_TOKEN_MORDINOS ":m"
+
+#define RADIO_CHANNEL_WRIGHTS "Wrights"
+#define RADIO_KEY_WRIGHTS "w"
+#define RADIO_TOKEN_WRIGHTS ":w"
+
+#define RADIO_CHANNEL_HOTEL "Hotel"
+#define RADIO_KEY_HOTEL "h"
```

Comment:
Do NOT use :h
That is the default for your department

Reply:
Just want to clarify, if it works, it works.
There may be serious unintended side effects tho